example/chrome-example.php

```diff
@@ -4,12 +4,37 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-// @TODO Change me
-$url = 'ws://127.0.0.1:9222/devtools/browser/b1c46ece-85c4-48b5-850d-687296d0566e';
 
-\Amp\Loop::run(function () use($url) {
+\Amp\Loop::run(function () {
     try {
-        $logger = new \Symfony\Component\Console\Logger\ConsoleLogger(new \Symfony\Component\Console\Output\ConsoleOutput(\Symfony\Component\Console\Output\OutputInterface::VERBOSITY_NORMAL));
+        $process = new \Amp\Process\Process(
+            [
+                "google-chrome-stable",
+                "--headless",
+                "--disable-gpu",
+                "--remote-debugging-port=9222",
+                "about:blank"
+            ]
+        );
+
+        $process->start();
+        $found = false;
+        $url = "";
+
+        while (!$found && $process->isRunning()) {
+            $readed = yield $process->getStderr()->read();
```

Comment:
I guess this should be `$readed .=`, apart from being named `$read` :stuck_out_tongue_winking_eye: